dist/stat.py

```diff
@@ -32,6 +34,39 @@ def check_unique_description(sorted_list):
             print("ERROR: repeated stat description in - '%s'" % (i.desc))
         temp = i.desc
 
+##########################################
+# Remove trailing digits for a string.
+# Used as a lambda in sorting stats by name.
+##########################################
+def remove_suffix_digits(str):
+    return re.sub(r'\d+$', '', str)
+    
+##########################################
+# Check names are sorted:
+# For each Stat subclass check the names are sorted in alphabetical order
+##########################################
+def check_name_sorted(stat_list):
+    stat_dict = defaultdict(list)
+    for stat in stat_list:
+        stat_dict[type(stat)].append(stat)
+    for stat_type, stats in stat_dict.items():
+        sorted_stats = sorted(stats, key=lambda stat: remove_suffix_digits(stat.name))
+        for i in range(len(stats)):
```

Comment:
I think we could use [zip()](https://docs.python.org/3/library/functions.html#zip) to traverse both lists at the same time.